contracts/state/StateV2.sol

```diff
@@ -42,10 +42,15 @@ contract StateV2 is Ownable2StepUpgradeable, IState {
     SmtLib.Data internal _gistData;
 
     /**
-     * @dev Network prefix
+     * @dev Default id type
      */
     bytes2 internal _defaultIdType;
 
+    /**
+     * @dev Transition Enabled For Onchain Identity
+     */
+    bool internal _transitionEnabledForOnchainIdentity;
```

Comment:
I like the idea of possibility to disable specific state transition methods (and maybe we need to have it too), but for the problem of uninitialized _defaultIdType I think it would be better to have `_defaultIdTypeInitialized` flag and moving it's check to `getDefaultIdType` from `transitStateGeneric`